### ISSUE TITLE
Is there any way to add/print custom step/text in html reports

### ISSUE THREAD

**Issue by durgeshshisode1988** [AOEpeople/geb-spock-reports#10]
Hi @tilmanginzel , 

I need your help to customize the reports if there is any provision. Please consider below scenario:
Scenario:
- I have below steps included in my spec file:

```
@Stepwise
class TC001_SMO_Scenario_Spec extends GebReportingSpec {
def "Step 1:Go to the login page of the WU"() {

        when: "User open the Login page"
        to LoginPage

        then: "Login page should get open"
        at LoginPage

        when: "User enters credentials"
        page.Login()

        then: "Home Page should get open"
        at HomePage
    }


    def "Step 2:User initiates transfer process"() {

        when: "Clicks on the Start New Transfer button"
        page.selectio()
        sleep(2000)

        then: "Reciept Number should be generated"
        at ManageConnections
    }
}
```

Now when the execution is done then below is the snapshot of report.

<img width="490" alt="gebreports" src="https://user-images.githubusercontent.com/39194204/40303632-2ca642d8-5d11-11e8-8d40-2210e49c3f31.PNG">

Now my requirement is that, there is some receipt number is getting generated on web page, I would like to have that to be printed in html report, after the Then block. (`then: "Reciept Number should be generated"`). 
Could you please guide me how this can be done with the help of specific geb/spock keywords or 
athaydes - spock-reports.

Thanks for your help on this.

**Comment by tilmanginzel** [comment]
Hi,

I'm quite busy the last few days, therefore the late response. 

I just saw that [spock-reports](https://github.com/renatoathaydes/spock-reports) 1.4.0 added a feature for this, called `featureInfo`. By using it you can add any info or text to the report you want. Unfortunately, my library and template does not yet support this neat feature, as it is still based on spock-reports 1.3.1. So for now, I don't know a good solution except trying to use the newest spock-reports version and updating the template by yourself.

Whenever I'll find the time to update this library, I will also make sure to upgrade the dependency to spock-reports, so that I can pick up all new features which are implemented there!

I don't know when I will find the time for it, hopefully it won't take too long.

**Comment by tilmanginzel** [comment]
I have just published version 0.2.0 which adds the `reportInfo` and `reportHeader` features from spock-reports.
